docs/tutorial/getting-started.md

```diff
@@ -31,14 +36,84 @@ Start off by deploying the jenkins-k8s charm. By default it will deploy the late
 of the jenkins-k8s charm.
 
 ```
-# Deploy an edge version of the charm until stable version is released.
 juju deploy jenkins-k8s --channel=latest/edge
 ```
 
+Wait for the charm to be active:
+```
+juju wait-for application jenkins-k8s
+```
+
 The Jenkins application can only have a single server unit. Adding more units through `--num-units`
 parameter will cause the application to misbehave.
 
-### Deploy and integrate agents
+
+### Expose jenkins-k8s through ingress
+
+Deploy traefik-k8s charm and integrate it with the jenkins-k8s charm:
+```
+juju deploy traefik-k8s --channel=latest/edge --trust
+juju integrate jenkins-k8s:ingress traefik-k8s
+```
+
+You can check the status with:
+```
+juju status --relations
+```
+
+After a few minutes, the deployment will be finished and all the units should be in 
+the active status.
+
+Run the following command to get the URL to connect to Jenkins:
+```
+juju run traefik-k8s/0 show-proxied-endpoints --format=yaml
+```
+
+The output will be something similar to:
+```
+Running operation 1 with 1 task
+  - task 2 on unit-traefik-k8s-0
+
+Waiting for task 2...
+traefik-k8s/0: 
+  id: "2"
+  results: 
+    proxied-endpoints: '{"traefik-k8s": {"url": "http://10.12.97.102"}, "jenkins-k8s":
+      {"url": "http://10.12.97.102/jenkins-tutorial-jenkins-k8s"}}'
+    return-code: 0
+  status: completed
+  timing: 
+    completed: 2024-09-27 15:09:36 +0200 CEST
+    enqueued: 2024-09-27 15:09:35 +0200 CEST
+    started: 2024-09-27 15:09:35 +0200 CEST
+  unit: traefik-k8s/0
+```
+
+In this case, the URL to use in your browser will be `http://10.12.97.102/jenkins-tutorial-jenkins-k8s`. In
```

Comment:
Can this URL be accessed from outside the Multipass VM?

Reply:
yes, traefik-k8s by default replies with the IP of the multipass machine, so it should work with and without multipass (that is why I chose traefik instead of the nging-integrator).